test/e2e/apimachinery/custom_resource_definition.go

```diff
@@ -251,6 +256,119 @@ var _ = SIGDescribe("CustomResourceDefinition resources [Privileged:ClusterAdmin
 		}
 	})
 
+	/*
+		Release : v1.17
+		Testname: Custom Resource Definition, defaulting
+		Description: Create a custom resource definition without default. Create CR. Add default and read CR until
+		the default is applied. Create another CR. Remove default, add default for another field and read CR until
+		new field is defaulted, but old default stays.
+	*/
+	framework.ConformanceIt("getting/updating/patching custom resource definition status sub-resource works ", func() {
```

Comment:
test title doesn't describe testing defaults, is that specifically what this is testing?

Reply:
copy&paste

Reply:
fixed